README.md

```diff
@@ -41,10 +46,25 @@ $ git submodule add https://github.com/getbraincloud/braincloud-unreal-plugin-sr
 
 Follow steps 4-10 above.
 
-### Unreal Engine 5.0
+### Unreal Engine Version
 
 The plugin will easily work with any version of UE4 and UE5. If switching, ensure that Intermediate files are cleaned up from previous build.
 
+### Platforms
+
+The plugin will work with all platforms. Make sure the required SDK, environment paths and licenses set up. To build for a specific platform only, edit the BCClient.plugin file to include:
+
+```
+"SupportedTargetPlatforms": [
+    "Win64",
+    "Android",
+    "Mac",
+    "IOS",
+    "Linux",
+    "LinuxArm64",
+    "Switch"
```

Comment:
We don't officially support switch just YET!

Reply:
YET!